src/Cli/dotnet/commands/dotnet-new/NewCommandParser.cs

```diff
@@ -43,6 +48,20 @@ internal static class NewCommandParser
             () => false,
             LocalizableStrings.DisableProjectContextEval_OptionDescription).Hide();
 
+        private static readonly Option<VerbosityOptions> s_verbosityOption = new(
+            new string[] { "-v", "--verbosity" },
+            () => DefaultVerbosity,
+            LocalizableStrings.Verbosity_OptionDescription)
+        {
+            ArgumentHelpName = CommonLocalizableStrings.LevelArgumentName
+        };
+
+
+        private static readonly Option<bool> s_diagnosticsOption = new(
+            new string[] { "-d", "--diagnostics" },
+            () => false,
+            Microsoft.DotNet.Tools.Help.LocalizableStrings.SDKDiagnosticsCommandDefinition);
+
```

Comment:
Had to create new instances of options - using those defined in SDK is not possible:
- diagnostic option doesn't have description, it is hardcoded in help handler instead
- verbosity option default value is quiet.